mozilla/kuma
# Bug 1197444: Rename kuma/static to kuma/assets

It's easy to confuse the kuma/static directory and the root static
directory, especially when the term kuma is already in the path.

Can you easily tell the difference between these?

    /path/to/dev/folder/kuma/kuma/static
    /path/to/dev/folder/kuma/static

Bleh. This change renames kuma/static to kuma/assets to make the
distinction more clear.

--

To test:

* Browse around and ensure the site generally looks good
* Repeat after [enabling production assets](https://kuma.readthedocs.org/en/latest/development.html#production-assets)
